pkg/util/config.go

```diff
@@ -37,3 +40,103 @@ func DefaultResourceThresholdStrategy() *slov1alpha1.ResourceThresholdStrategy {
 		MemoryEvictThresholdPercent: pointer.Int64Ptr(70),
 	}
 }
+
+func DefaultMemoryQOS(qos apiext.QoSClass) *slov1alpha1.MemoryQOS {
```

Comment:
can you add some explanations for these default values?

Reply:
done